arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -827,52 +838,52 @@
           ]
         },
         "upgradeMode": {
-          "description": "The cluster upgrade mode",
+          "description": "Cluster upgrade mode indicates if fabric upgrade is initiated automatically by the system or not",
           "type": "string",
           "enum": [
             "Automatic",
             "Manual"
           ]
         },
         "clientCertificateThumbprints": {
-          "description": "The client certificate thumbprints",
+          "description": "The client thumbprint details ,it is used for client access for cluter operation",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ClientCertificateThumbprint"
           }
         },
         "clientCertificateCommonNames": {
-          "description": "The certificate common name and issurer thumprint for the client certificate",
+          "description": "Client certificate details using common name",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ClientCertificateCommonName"
           }
         },
         "fabricSettings": {
-          "description": "The faric setting collection",
+          "description": "ServiceFabric section settings",
           "type": "array",
           "items": {
             "$ref": "#/definitions/SettingsSectionDescription"
           }
         },
         "reverseProxyCertificate": {
-          "description": "The certificate for the reverse proxy",
+          "description": "The certificate details for the reverse proxy",
           "$ref": "#/definitions/CertificateDescription"
         },
         "managementEndpoint": {
-          "description": "The cluster management endpoint address",
+          "description": "Http endpoint for client access",
           "type": "string"
         },
         "nodeTypes": {
-          "description": "The arary of the nodeType",
+          "description": "The list of nodetypes that make up the cluster",
           "type": "array",
           "items": {
             "$ref": "#/definitions/NodeTypeDescription"
           }
         },
         "provisioningState": {
           "readOnly": true,
-          "description": "The provisioning state of the cluster",
+          "description": "The provisioning state of the cluster resource",
```

Comment:
Please explain in description what is meant by Default provisioning state, if a user need to put a switch block for provisioning state that will guide him to code against case where provisioningState is Default.